velodrome/config.yaml

```diff
@@ -100,6 +100,11 @@ projects:
             static_configs:
               - targets:
                   - submit-queue.k8s.io
+          - job_name: kubernetes-boskos
+            metrics_path: /prometheus
+            static_configs:
+              - targets:
+                  - http://35.192.170.153:8080
```

Comment:
There is a `boskos-status` service in the `test-pods` namespace of the `k8s-prow-builds` project's `prow` cluster.